components/script/dom/htmlinputelement.rs

```diff
@@ -1162,6 +1159,11 @@ impl Validatable for HTMLInputElement {
         }
         true
     }
+    fn validate(&self, validate_flags: ValidationFlags) -> bool {
+        if validate_flags.is_empty() {}
+        // Need more flag check for different validation types later
```

Comment:
returning only true in here, leaves unused variable names warning. Any ideas or we can avoid that for time being?